### DISCUSSION TITLE
position : absolute not working in react-vite

### DISCUSSION THREAD

**Discussion by pranavmappoli** [tailwindlabs/tailwindcss#15258] [Help]
**What version of Tailwind CSS are you using?**

 "tailwindcss": "^3.4.15",

**What build tool (or framework if it abstracts the build tool) are you using?**

vite
**What version of Node.js are you using?**

For example: v18

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS, Windows

**Reproduction URL**
https://github.com/pranavmappoli/frontend-system-design/tree/main/Popover

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

**Comment by pranavmappoli** [comment]
Same works fine when i downgrade to "tailwindcss": "^3.4.5",

**Comment by adamwathan** [comment]
Looks like you need to update your `content` configuration to include the `Popover` folder:

https://github.com/pranavmappoli/frontend-system-design/blob/main/Popover/tailwind.config.js#L3
